python/pyspark/cloudpickle.py

```diff
@@ -72,6 +78,22 @@
     PY3 = True
```

Comment:
Is this reversed? looks like this is meant to be true when running Python 3? (I know you didn't change it here)

Reply:
> Is this reversed? looks like this is meant to be true when running Python 3? (I know you didn't change it here)

Can you state it more precisely? What I see is `PY3 = True` is executed when `sys.version < '3'` is False --- that's OK.

Reply:
It's probably my misunderstanding, but sys.version < '3' on Python 2 right? PY3 seems like it indicates whether Python 3 is running?

Reply:
Ugh, nevermind, this is github folding out intermediate code.